### PR TITLE
[13.0][MIG] event_sale

### PR DESCRIPTION
**Pull request by bishalgit**
Nothing to do

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by MiquelRForgeFlow — APPROVED** [review]

File: addons/event_sale/migrations/13.0.1.1/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---Models in module 'event_sale'---
+new model event.event.configurator [transient]
+# NOTHING TO DO
+
+---Fields in module 'event_sale'---
+event_sale   / event.registration       / campaign_id (many2one)        : NEW relation: utm.campaign, isrelated: related, stored
+event_sale   / event.registration       / medium_id (many2one)          : NEW relation: utm.medium, isrelated: related, stored
+event_sale   / event.registration       / source_id (many2one)          : NEW relation: utm.source, isrelated: related, stored
+# NOTHING TO DO: Filled by standard update mechanism being a related. Possible improvement: being filled by SQL for performance.
```
Comment (pedrobaeza) [inline addons/event_sale/migrations/13.0.1.1/openupgrade_analysis_work.txt:9]: The filling by SQL is already done also by core after a recent update.

File: addons/event_sale/migrations/13.0.1.1/openupgrade_analysis_work.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+NEW ir.ui.view: event_sale.assets_tests
+NEW ir.ui.view: event_sale.qunit_suite
+NEW ir.ui.view: event_sale.event_configurator_view_form
+DEL ir.ui.view: event_sale.assets_backend_inherit_event_sale
+DEL ir.ui.view: event_sale.view_order_form_inherit_event_sale_group_event_user
+# NOTHING TO DO
+
+NEW ir.ui.view: event_sale.sale_order_view_form
+DEL ir.ui.view: event_sale.view_sale_order_form_inherit_event
+# NOTHING TO DO: It is rename but there is no xml that inherits old xml.
```
Comment (pedrobaeza) [inline addons/event_sale/migrations/13.0.1.1/openupgrade_analysis_work.txt:24]: It's better to rename it anyway for avoiding third-party dangling inherits

Comment (jcdrubay) [reply addons/event_sale/migrations/13.0.1.1/openupgrade_analysis_work.txt:24]: Like that?

in `pre-migration.py`

```
from openupgradelib import openupgrade


@openupgrade.migrate()
def migrate(env, version):

    xmlids_spec = [
        (
            "event_sale.view_sale_order_form_inherit_event",
            "event_sale.sale_order_view_form"
        ),
    ]
    openupgrade.rename_xmlids(cr, xmlids_spec)
```

**Comment by MiquelRForgeFlow** [comment]
Superseded by https://github.com/OCA/OpenUpgrade/pull/2724.

**Pull request closed without merging**